Sources/Bodega/SQLiteStorageEngine.swift

```diff
@@ -211,6 +211,7 @@ public actor SQLiteStorageEngine: StorageEngine {
     /// Removes all the `Data` items from the database.
     public func removeAllData() throws {
         try self.connection.run(Self.storageTable.delete())
+        let _ = try? self.connection.vacuum()
```

Comment:
@mikakruschel It appears that `vacuum()` is annotated as `@ discardableResult`, which means we can safely shorten this to `try? self.connection.vacuum()`. Would you be able to make that small change?